src/xspf/xspf.c

```diff
@@ -412,7 +413,21 @@ static gboolean xspf_playlist_save (const gchar * filename, VFSFile * file,
                 if (isOK)
                     xspf_add_node(track, xs->type, xs->isMeta, xs->xspfName, scratch, scratchi);
             }
+	    scratch = tuple_get_str (tuple, FIELD_MBID);
+	    if (scratch != NULL)
+	    {
+		xmlNodePtr musicbrainz;
+		gchar tmps[94];
+
+		musicbrainz = xmlNewNode(NULL, (xmlChar *) "meta");
+		xmlSetProp(musicbrainz, (xmlChar *) "rel", (xmlChar *) "http://musicbrainz.org/recording");
+		g_snprintf(tmps, sizeof(tmps),
+			   "http://musicbrainz.org/ws/2/recording/%s?inc=artist-credits", scratch);
+		xmlAddChild (musicbrainz, xmlNewText ((xmlChar *) tmps));
+		xmlAddChild (track, musicbrainz);
```

Comment:
What does `track` to here? If it's supposed to use MusicBrainz terminology for the MBID, then it should be `recording` (as also per the two links just above).